src/main/java/com/chariotsolutions/meetupnotifier/EventIdRow.java

```diff
@@ -0,0 +1,20 @@
+package com.chariotsolutions.meetupnotifier;
+
+public class EventIdRow {
```

Comment:
Please rename this to `CalendarEvent` or `CalendarEventKey`, depending on what you rename the table to.